package-system/AWSNativeSDK/README.md

```diff
@@ -0,0 +1,101 @@
+# Build New AWS Service Target in AWSNativeSDK
+First check package directories at [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp) on GitHub. 
+The package name is the suffix of the directory name for the service.
+
+```
+aws-sdk-cpp\aws-cpp-sdk-<packageName>   # Repo directory name and packageName
+aws-sdk-cpp\aws-cpp-sdk-s3              # Example: Package name is s3
+```
+
+Note: If your target service is not available in current AWSNativeSDK version (but it is listed in aws-sdk-cpp repo),
```

Comment:
Nit: When we mention AWSNativeSDK thats really an internal term, consider linking to what this is or add a clarifying statement.

Reply:
Maybe also "If the AWS service packge is not currently used in AWSNativeSDK, then..."

Reply:
done